web/src/components/PoisMobile.tsx

```diff
@@ -180,6 +185,7 @@ const PoisMobile = ({
           </GoBackContainer>
         )}
         <ListContainer>
+          {!canGoBack ? <ListTitle>{t('listTitle')}</ListTitle> : undefined}
```

Comment:
You can use the logical AND here as a shorthand like this: `{!canGoBack && <ListTitle>{t('listTitle')}</ListTitle>}`. But it's also fine like this, your call.